cognite_toolkit/cdf_tk/describe.py

```diff
@@ -0,0 +1,162 @@
+        table.add_column("Info", justify="left")
+        table.add_column("Value", justify="left", style="green")
+        table.add_row("Number of properties", str(len(view.properties)))
+        table.add_row("Used for", str(view.used_for))
+
+        implements_str = [f"{i}\n" for i in view.implements or []]
+        if len(implements_str) > 0:
+            implements_str[-1] = implements_str[-1][0:-1]
+        table.add_row("Implements", "".join(implements_str))
+        properties = [f"{p}\n" for p in view.properties.keys()]
+        if len(properties) > 0:
+            properties[-1] = properties[-1][0:-1]
+        table.add_row("List of properties", "".join(properties))
+        direct_relations_str = []
+        edge_relations_str = []
+        nr_of_direct_relations = 0
+        nr_of_edge_relations = 0
+        for p, edge_type in view.properties.items():
+            if type(edge_type.type) is DirectRelation:
+                nr_of_direct_relations += 1
+                if edge_type.source is None:
+                    direct_relations_str.append(f"{p} --> no source")
+                    continue
+                direct_relations_str.append(
+                    f"{p} --> ({edge_type.source.space}, {edge_type.source.external_id}, {edge_type.source.version})\n"
+                )
+            elif type(edge_type.type) is DirectRelationReference:
+                nr_of_edge_relations += 1
+                edge_relations_str.append(
+                    f"{p} -- {edge_type.direction} --> ({edge_type.source.space}, {edge_type.source.external_id}, {edge_type.source.version})\n"
+                )
+        if len(direct_relations_str) > 0:
+            direct_relations_str[-1] = direct_relations_str[-1][0:-1]
+        if len(edge_relations_str) > 0:
+            edge_relations_str[-1] = edge_relations_str[-1][0:-1]
+        table.add_row(f"Direct relations({nr_of_direct_relations})", "".join(direct_relations_str))
+        table.add_row(f"Edge relations({nr_of_edge_relations})", "".join(edge_relations_str))
+        node_count = 0
+        # Iterate over all the nodes in the view 1,000 at the time
+        try:
+            result = client.data_modeling.instances.aggregate(
+                view.as_id(),
+                aggregates=Count("externalId"),
+                instance_type="node",
+            )
+            node_count = int(result.value)
+        except Exception as e:
+            print(f"Failed to retrieve nodes for view {view.external_id} version {view.version} in space {view.space}.")
+            print(e)
+        table.add_row("Number of nodes", f"{node_count:,}")
+        edge_count = 0
+        # Iterate over all the edges in the view 1,000 at the time
+        try:
+            result = client.data_modeling.instances.aggregate(
+                view.as_id(),
+                aggregates=Count("externalId"),
+                instance_type="edge",
+            )
```

Comment:
This is a bug, edges are not located in views (unless they have properties).

Reply:
I realized I do not know how to do this so I have asked in the topic-data-modeling channel.